src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -226,7 +214,6 @@ public void run() {
     }
     clients.clear();
     servers.clear();
-    dogCache.cleanAll();
```

Comment:
I think its fine, there could be a need to try and purge it all at some point but its really not needed here